### ISSUE TITLE
Cropper support

### ISSUE THREAD

**Issue by ryanswrt** [TheAncientGoat/autoform-slingshot#6]
See https://github.com/pfafman/meteor-photo-up/

Turn preview widget into cropper or have a modal popup

Cropper action must happen before onBefore callback

**Comment by timbrandin** [comment]
Here's another solution: http://fengyuanchen.github.io/cropper/

**Comment by timbrandin** [comment]
When on a mobile, this would be a nice experience: https://developers.aviary.com/
I'm not sure if aviary is web-capable, but it'd definitely be amazing if it was.

I've done some cropping tools before actually, it would be nice to have it support both bootstrap and ionic + web and mobile at the same time.

**Comment by demiantriebl** [comment]
i try to put on onBefore callback but i can display the image to crop, how can do it?
